style.css

```diff
@@ -127,26 +196,7 @@ body {
   flex: none;
   order: 2;
   flex-grow: 0;
-}
-
-.social-media {
-  margin-left: 2em;
-  margin-right: 24 px;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  padding: 0;
-  width: 13em;
-  flex: none;
-  order: 3;
-  flex-grow: 0;
-  text-decoration: none;
-}
-
-.social-media > li {
-  list-style: none;
-  text-decoration: none;
+  max-width: 50vh;
 }
 
 .works {
```

Comment:
I must tell you very kindly and respectfully that what you want here is not possible to do using only html and css.
To change an entire card as I have to apply "display: none" to the same card when I hover over it, then the card disappears and I have to try to change the background image or replace the card itself.
To change its properties when I mouse over it I have to use javascript, I already thought about it, I researched it for more than 3 hours to be sure but it's not possible.
On the other hand, I consider that the objective of this week was to learn things about html and css and to demonstrate that one has learned to use html and css, to copy or imitate a web design as well as possible, so the use of js is not available this week.

Reply:
Hi @mrcbq,

You've have spoken quite well. However, considering the fact that a target such as `.card:hover .shown` is possible, I think this can be achieved with CSS. The selection above can style a child class "shown" when the parent class "card" is hovered.

Reply:
Hello, @emmiiorji  thank you for responding to the student. @mrcbq  kindly refer to the comment I left below on how to resolve this.

#happycoding :rocket: